src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -15589,7 +15589,54 @@ def pct_change(
                 ).frame
             )
 
-    def equals(self, other: "SnowflakeQueryCompiler") -> "SnowflakeQueryCompiler":
+    def index_equals(self, other: "SnowflakeQueryCompiler") -> bool:
```

Comment:
Aren't we keeping docstrings under the `docstrings` folder for Index methods? cc: @sfc-gh-vbudati

Reply:
No, that change has not been made yet @sfc-gh-helmeleegy. All index docstrings are currently in the same file as the code.

Reply:
I see. Thanks.